<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsPingAssistant</class>
 <widget class="QWidget" name="xToolsPingAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutStatusBar"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutToolBar"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
